src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -36,15 +39,35 @@ private static void init(){}
         public static final TagKey<Block> BARRELS = tag("barrels");
         public static final TagKey<Block> BARRELS_WOODEN = tag("barrels/wooden");
         public static final TagKey<Block> BOOKSHELVES = tag("bookshelves");
+        public static final TagKey<Block> BUDDING_BLOCKS = tag("budding_blocks");
+        public static final TagKey<Block> BUDS = tag("buds");
         public static final TagKey<Block> CHESTS = tag("chests");
         public static final TagKey<Block> CHESTS_ENDER = tag("chests/ender");
         public static final TagKey<Block> CHESTS_TRAPPED = tag("chests/trapped");
         public static final TagKey<Block> CHESTS_WOODEN = tag("chests/wooden");
+        public static final TagKey<Block> CLUSTERS = tag("clusters");
         public static final TagKey<Block> COBBLESTONE = tag("cobblestone");
         public static final TagKey<Block> COBBLESTONE_NORMAL = tag("cobblestone/normal");
         public static final TagKey<Block> COBBLESTONE_INFESTED = tag("cobblestone/infested");
         public static final TagKey<Block> COBBLESTONE_MOSSY = tag("cobblestone/mossy");
         public static final TagKey<Block> COBBLESTONE_DEEPSLATE = tag("cobblestone/deepslate");
+        public static final TagKey<Block> DYED_BLOCKS = tag("dyed_blocks");
```

Comment:
Why `dyed_blocks` instead of just `dyed`? Partity with the item tag seems more important to me than including the registry name in the tag name.

Reply:
Only issue I have with removing `_blocks` is the base tag will then become just `dyed.json` which might seem odd as a tag name. Or is that a preferable name actually?